README.md

```diff
@@ -1,2 +1,36 @@
-# BIP39Wallet-NBitcoin
-# BIP39Wallet-NBitcoin
+# BIP39 wallet for AElf unity SDK
+## Introduction
+This is a BIP39 wallet project for AElf unity SDK. It can generate mnemonic words and private and public key, and can also import mnemonic words to get private key. It can also generate address from public key.
```

Comment:
replaced this whole sentence by "It can generate mnemonic words, private-public keypair and wallet address, and can also import mnemonic words to get the associated keypair and address."